src/openforms/contrib/customer_interactions/client.py

```diff
@@ -39,3 +45,42 @@ def get_digital_addresses_for_bsn(self, bsn: str) -> Iterator[DigitaalAdres]:
         }
         response = self.digitaal_adres.list_iter(params=params)
         yield from response
+
+    def create_customer_contact(
+        self, submission: Submission
+    ) -> MaakKlantContactResponse:
+        data: MaakKlantContactCreateData = {
+            "klantcontact": {
+                "kanaal": "open forms",
```

Comment:
can we use a more generic name instead of a product name here?

Something like "Webformulier" probably makes more sense?